### PR TITLE
Remove mem leak

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
On the client side the EventStream is closed automatically, not sure it's the same on server side. 

Now on connection closed the listener is removed and #17 no longer appears on multiple refreshes.  Not sure how this works with multiple users on the site at the same time? **Needs testing**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -20,8 +20,14 @@ app.get("/subscribe", (req, res) => {
...
 
-  Stream.on("push", (event, data) => {
+  function onPush(event, data) {
     res.write(`data: ${data} \n\n`);
+  }
+
+  Stream.on("push", onPush);
+
+  req.on("close", function() {
+    Streamt.removeListener("push", onPush);
```
Comment (wanecek) [inline app.js]: Typo here (`s/Streamt/Stream`)

Comment (wanecek) [reply app.js]: Yeah, that'd be good :)

**Comment by wanecek** [comment]
Pushed minor changes to procastinate tentaplugg, PTAL @OskarDamkjaer :)

**Comment by OskarDamkjaer** [comment]
Nice fixes! I think the rest of the project uses double quotes though, ran prettier and merged :)

**Pull request merged**